pennylane_lightning/lightning_tensor/_tensornet.py

```diff
@@ -143,8 +143,8 @@ class LightningTensorNet:
             be described but the larger the memory requirements. Note that GPUs are ill-suited (i.e. less
             competitive compared with CPUs) for simulating circuits with low bond dimensions and/or circuit
             layers with a single or few gates because the arithmetic intensity is lower.
-        cutoff (float): The threshold used to truncate the singular values of the MPS tensors. The default is 0.
-        cutoff_mode (str): Singular value truncation mode for MPS tensors. The options are ``"rel"`` and ``"abs"``. The default is ``"abs"``.
+        cutoff (float): The threshold used to truncate the singular values of the MPS tensors. Default is 0.
+        cutoff_mode (str): Singular value truncation mode for MPS tensors. Options:[``"rel"``, ``"abs"`` (default)].
```

Comment:
```suggestion
        cutoff_mode (str): Singular value truncation mode for MPS tensors can be done either by
            considering the absolute values of the singular values (``"abs"``) or by considering
            the relative values of the singular values (``"abs"``). Default is ``"abs"``.
```

Reply:
`considering the absolute values of the singular values (``"abs"``) or by considering the relative values of the singular values (``"rel"``).` @maliasadi